spec/classes/init_spec.rb

```diff
@@ -513,15 +513,15 @@
         it { is_expected.to contain_concat_fragment('squid_https_port_2001').with_content(%r{^https_port\s+2001\s+special for 2001$}) }
       end
 
-      if facts['osfamily'] == 'RedHat'
+      if facts[:osfamily] == 'RedHat'
         context 'with http_port parameters set + SELINUX' do
           let :params do
             { config: '/tmp/squid.conf',
               http_ports: { 2000 => { 'options' => 'special for 2000' } } }
           end
           let(:facts) do
             facts.merge(
-              selinux => true
+              selinux: true
```

Comment:
This sets the legacy fact. See https://github.com/voxpupuli/voxpupuli-test#fact-handling for how we have `override_facts` to correctly set modern structured facts.